docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -173,12 +173,21 @@ for customer data including session recordings and user records.
 
 ## Can I get a list of IP addresses to configure a firewall?
 
-We don't currently offer a list of addresses for firewall configuration and plan to make this available in the future.
+We do not have plans to offer a list of IP addresses for Teleport Enterprise
+Cloud at the moment.
 
-## Can I configure Teleport Enterprise Cloud to only allow connections from an allowlist of IP addresses?
+We do provide customers with a stable tenant DNS name backed with a TLS
+certificates from [letsencrypt.org](https://letsencrypt.org) that can be
+utilized to verify the integrity of any outbound connections to Teleport
+Enterprise Cloud.
 
-No. We believe mTLS with strong identity is the best standard available for authenticating and access clients.
-We are also investigating additional and new features that can be used as a defense in depth strategy for use with Teleport Enterprise Cloud in 2022.
+## Can I configure an allowlist of inbound IP addresses to Teleport Enterprise Cloud?
+
+We do not have plans to offer support for inbound connection IP allowlists.
+
+We believe mTLS with strong user and [device
```

Comment:
this should be inline link url `[device identity](../../access-controls/guides/device-trust.mdx)`